### PR TITLE
HagelTechnologiesLtd.DUMeter version 8.01

### PR DESCRIPTION
**Pull request by TechnicaVivunt**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/55736)

### CODE REVIEW

**Comment by TechnicaVivunt** [comment]
this commit closes #55573

File: manifests/h/HagelTechnologiesLtd/DUMeter/8.01/HagelTechnologiesLtd.DUMeter.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: HagelTechnologiesLtd.DUMeter
+PackageVersion: 8.01
+PackageLocale: en-US
+Publisher: Hagel Technologies Ltd.
+PackageName: Du Meter
+License: Copyright © 1997-2022 Hagel Technologies Ltd.
```
Comment (Trenly) [inline manifests/h/HagelTechnologiesLtd/DUMeter/8.01/HagelTechnologiesLtd.DUMeter.locale.en-US.yaml:9]: A copyright is not a license 

```suggestion
License: Proprietary
LicenseUrl: https://www.hageltech.com/dumeter/license
Copyright: Copyright © 1997-2022 Hagel Technologies Ltd.
CopyrightUrl: https://www.hageltech.com/dumeter/license
```

File: manifests/h/HagelTechnologiesLtd/DUMeter/8.01/HagelTechnologiesLtd.DUMeter.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: HagelTechnologiesLtd.DUMeter
+PackageVersion: 8.01
+PackageLocale: en-US
+Publisher: Hagel Technologies Ltd.
+PackageName: Du Meter
```
Comment (Trenly) [inline manifests/h/HagelTechnologiesLtd/DUMeter/8.01/HagelTechnologiesLtd.DUMeter.locale.en-US.yaml:8]:
```suggestion
PackageName: DU Meter
```

File: manifests/h/HagelTechnologiesLtd/DUMeter/8.01/HagelTechnologiesLtd.DUMeter.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: HagelTechnologiesLtd.DUMeter
+PackageVersion: 8.01
+Installers:
+- InstallerUrl: https://dl.hageltech.com/DUMeter-Install.exe
+  Architecture: neutral
+  InstallerType: inno
+  InstallerSha256: DAB43C03A5967284DBB8C5CC975AAB5F06542439957463291A55A83D9C4C5845
```
Comment (Trenly) [inline manifests/h/HagelTechnologiesLtd/DUMeter/8.01/HagelTechnologiesLtd.DUMeter.installer.yaml:10]:
```suggestion
  InstallerSha256: DAB43C03A5967284DBB8C5CC975AAB5F06542439957463291A55A83D9C4C5845
  ProductCode: 'DUMeter3_is1'
```

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by TechnicaVivunt** [comment]
Committed Reflected Changes. Will double check in the morning to see if any further adjustments will need to be made.

**Review by ItzLevvie — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**